### ISSUE TITLE
Caraka Siddhisthāna 12 sub-section numbering

### ISSUE THREAD

**Issue by gennaios** [sarit/SARIT-corpus#122]
Hello,

Apologies if I am in error. I have not consulted the used edition as I do not have it though I may look for it. I am using the edition of P.V. Sharma as well as another, both derivatives. Is it possible that the sub-subsections are mis-numbered? A small detail.

**Comment by wujastyk** [comment]
You'll have to be more specific: can you point to the passages you are seeing?

The edition of Caraka that we use is available here: [https://archive.org/details/TheCarakasahitOrCharakasamhitaOfCarakaEditedByYadavaarman](https://archive.org/details/TheCarakasahitOrCharakasamhitaOfCarakaEditedByYadavaarman)

You could clone the repository, make the corrections yourself, and then create a pull request.  That would be the very best way to help the project!  :-)  

Best regards, and thanks for your engagement!

Dominik

**Comment by gennaios** [comment]
Dominik,

Thank you for the PDF link.

I confess I am at amateur at Sanskrit. My studies are only at their beginning, and for the moment I remain sometimes or often distracted not able to concentrate on things that take sustained effort.

It is the subdivision numbering. Elsewhere, at least where I've looked, they are numbered from the first, e.g. Ca.2.1.12-1, Ca.2.1.12-2, etc., yet for Si.12, for each subdivision the first is unnumbered while the second begins as 1, e.g. Ca.8.12.14, Ca.8.12.14-1, Ca.8.12.14-2, …. I compared a few and confirmed ; perhaps such is the case for all of Si.12. Subdivisioned verses are : 14, 15, 16, 18, and 19.

I may be able to get to it sometime. I don't at all mind helping out. I think you understand what is at stake is great. Perhaps Ayurveda – availability, translation, and understanding of texts –, of this or some other medicine, is only some small part. Perhaps one could say we are no longer thinking, or feeling, or observing in any way that resembles the tradtions that survive or that still live today. The world is not as such where such naturally and spontaneously comes into being. As such, who is able to examine such traditions and truly see and feel like those that created them ? I do not know. … What I am trying to do lies elsewhere. More on that later. :)
